### PR TITLE
Cardview

### PR DESCRIPTION
**Pull request by elegant651**
I've done to eventlist bind with server API. And made new component as cameraview. 
I also adjust to configuration for iosapp.

### CODE REVIEW

File: __tests__/list-test.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+// import EventList from '../shared/components/EventList';
+
+// Note: test renderer must be required after react-native.
+// import renderer from 'react-test-renderer';
+
+it('renders correctly', () => {
+  // const tree = renderer.create(
+  //   <EventList />
+  // ).toJSON();
+  // expect(tree).toMatchSnapshot();
```
Comment (ochanje210) [inline __tests__/list-test.js]: why did u commented out the test case?

File: android/app/src/main/java/com/goober/MainApplication.java [hunk 2]
```diff
@@ -25,6 +26,7 @@ protected boolean getUseDeveloperSupport() {
     protected List<ReactPackage> getPackages() {
       return Arrays.<ReactPackage>asList(
           new MainReactPackage(),
+            new RCTCameraPackage(),
```
Comment (ochanje210) [inline android/app/src/main/java/com/goober/MainApplication.java]: is this proper indent

File: __tests__/list-test.js (hunk 1 above)
Comment (elegant651) [inline __tests__/list-test.js]: it's not work properly. 'fetch' is not work in this test_case. I should figure out the issue.

File: android/app/src/main/java/com/goober/MainApplication.java (hunk 2 above)
Comment (elegant651) [inline android/app/src/main/java/com/goober/MainApplication.java]: I think it attached automatically when the library has installed. you can resolve it.

**Comment by ochanje210** [comment]
LGTM

**Comment by elegant651** [comment]
LGTM

**Pull request merged**